src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -928,14 +948,14 @@
     (fs/exists? hostpubkey)
     (throw
      (IllegalStateException.
-      (i18n/trs "Found master public key ''{0}'' but master private key ''{1}'' is missing"
+      ^String (i18n/trs "Found master public key ''{0}'' but master private key ''{1}'' is missing"
```

Comment:
Editor was complaining that this was ambiguous, so I specified the type explicitly.